From 8bcd1e8a7932ccdc140f002177f24b43aa84fca6 Mon Sep 17 00:00:00 2001
From: Lars Wendler <polynomial-c@gentoo.org>
Date: Sat, 4 Jun 2016 19:57:48 +0200
Subject: [PATCH 2/5] cpupower.c: Increase max CPUs to 16.

A better fix would be to make this a configuration item in the GUI.

Signed-off-by: Lars Wendler <polynomial-c@gentoo.org>
---
 cpupower.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cpupower.c b/cpupower.c
index d4e2fab..ccc72d9 100644
--- a/cpupower.c
+++ b/cpupower.c
@@ -65,7 +65,7 @@
 */
 #define  STYLE_NAME	"cpupower"
 
-#define NCPU_MAX 8
+#define NCPU_MAX 16
 
 static unsigned int ncpu;
 static unsigned long khz_max;
-- 
2.8.3

